test.py

```diff
@@ -9,8 +11,17 @@
 
 print oc.toRadString()
 
+
+genday = Gendaymtx(v=True,h=True)
+genday.inputFiles =[r'C:\Ladybug\annual\annualSimulation\State_College__Penn_State___Su_PA_USA.wea'] #This path needs to be present physically !
```

Comment:
@mostaphaRoudsari This was already in place. The mistake that I made in the test file is to specify the input through .inputFiles. I should have done that through .weaFile. Can you either make inputFiles to be private or raise an error when they are set directly ( like I did there) ?